src/middlewares/countryNameValidation.js

```
@@ -0,0 +1,24 @@
+const { getCountriesData } = require('~/utils/getLocationsApiData')
+const { createError } = require('~/utils/errorsHelper')
+const { NOT_FOUND, BAD_REQUEST } = require('~/consts/errors')
+
+const countryNameValidation = () => {
+  return async (req, _res, next) => {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)